samples/sensor/grow_r502a/src/main.c

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2021 Linumiz
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr.h>
+#include <device.h>
+#include <drivers/sensor.h>
+#include <drivers/uart.h>
+#include <drivers/gpio.h>
+
+#define FPS_DEV_NAME	DT_LABEL(DT_INST(0, hzgrow_grow_r502a))
```

Comment:
Please don't use DT_INST in samples; that can be avoided with DEVICE_DT_GET_... as described elsewhere in my review comments. The device structure has a `name` field you can use for this.

Reply:
thanks for the reviews, Fixed this and all the above reviews